src/main/resources/application.properties

```diff
@@ -0,0 +1,32 @@
+spring.datasource.url =
+spring.datasource.username = 
+spring.datasource.password =
+
+## Keep the connection alive if idle for a long time (needed in production)
+spring.datasource.testWhileIdle = true
+spring.datasource.validationQuery = SELECT 1
+
+# Show or not log for each sql query
+spring.jpa.show-sql = true
+
+# Hibernate properties
+## Use spring.jpa.properties.* for Hibernate native properties (the prefix is
+## stripped before adding them to the entity manager)
+## Hibernate ddl auto (create, create-drop, update, validate)
+spring.jpa.hibernate.ddl-auto = create
+# Naming strategy
+spring.jpa.hibernate.naming-strategy = org.hibernate.cfg.ImprovedNamingStrategy
+## The SQL dialect makes Hibernate generate better SQL for the chosen database
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
```

Comment:
Could this be made compatible with PostgreSQL? That is the production database so it makes sense to use the same in development.